fedimint-server/src/net/p2p_connector.rs

```diff
@@ -179,7 +179,9 @@ where
             .with_single_cert(vec![certificate], self.cfg.private_key.clone())
             .expect("Failed to create TLS config");
 
-        let listener = TcpListener::bind(self.p2p_bind_addr).await?;
+        let listener = TcpListener::bind(self.p2p_bind_addr)
+            .await
+            .expect("Could not bind to port");
```

Comment:
Philosophical question: do we want to panic here or bubble up such errors to the top level ideally? @dpc I'm not opposed to this approach, but maybe I'm missing something.